apps/web/src/common/modules/widgets/DailyUpdates.vue

```diff
@@ -14,33 +14,31 @@
         </template>
 
         <template #default>
-            <div v-if="loading"
-                 class="overflow-hidden"
+            <p-data-loader :loading="loading"
+                           :data="data"
+                           loader-type="skeleton"
+                           class="card-wrapper"
+                           :class="{'fixed-height': loading || !data.length}"
             >
-                <div v-for="v in skeletons"
-                     :key="v"
-                     class="flex p-4 items-center"
-                >
-                    <p-skeleton width="2rem"
-                                height="2rem"
-                                class="mr-4 flex-shrink-0"
-                    />
-                    <div class="grid grid-cols-1 gap-1 w-full">
-                        <p-skeleton width="80%"
-                                    height="0.625rem"
-                        />
-                        <p-skeleton width="100%"
-                                    height="0.625rem"
+                <template #loader>
+                    <div v-for="v in skeletons"
+                         :key="`skeleton-${v}`"
+                         class="flex p-4 items-center"
+                    >
+                        <p-skeleton width="2rem"
+                                    height="2rem"
+                                    class="mr-4 flex-shrink-0"
                         />
+                        <div class="grid grid-cols-1 gap-1 w-full">
+                            <p-skeleton width="80%"
+                                        height="0.625rem"
+                            />
+                            <p-skeleton width="100%"
+                                        height="0.625rem"
+                            />
+                        </div>
                     </div>
```

Comment:
suggestion: You can give parent <div> to prevent customizing css with deep. :)

Reply:
Due to time constraints, I will refactor it next time!🥲